packages/store-indexer/src/sqlite/apiRoutes.ts

```diff
@@ -7,10 +9,9 @@ import { debug } from "../debug";
 import { createBenchmark } from "@latticexyz/common";
 import { compress } from "../koa-middleware/compress";
 import { getTablesWithRecords } from "./getTablesWithRecords";
-import { BaseSQLiteDatabase } from "drizzle-orm/sqlite-core";
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function apiRoutes(database: BaseSQLiteDatabase<"sync", any>): Middleware {
+export function apiRoutes(database: BaseSQLiteDatabase<"sync", any>, enableQueryEndpoint = false): Middleware {
```

Comment:
small nit but would turn this into an object now that we have more than one thing
```suggestion
export function apiRoutes({ database, enableUnsafeQueryApi = false }: { database: BaseSQLiteDatabase<"sync", any>, enableUnsafeQueryApi?: boolean }): Middleware {
```